Xamarin.Forms.Xaml/XmlnsHelper.cs

```diff
@@ -46,5 +59,17 @@ public static void ParseXmlns(string xmlns, out string typeName, out string ns,
 					typeName = decl;
 			}
 		}
+
+		static void ParseUsing(string xmlns, out string typeName, out string ns, out string asm, out string targetPlatform)
+		{
+			typeName = ns = asm = targetPlatform = null;
+
+			foreach (var decl in xmlns.Split(';')) {
```

Comment:
So if the user does something silly, like include more than one `using`, then the last one wins. Is this something we need to have a test for to establish the behavior as expected?